InvenTree/common/files.py

```diff
@@ -109,6 +111,33 @@ def setup(self):
             # Update headers
             self.update_headers()
 
+        # TODO maybe not here but in an own function?
```

Comment:
How do you see this? Like `setup_order` and `setup_part` methods? I think I prefer to keep the `setup` method generic and re-using `self.name`.

Reply:
Makes sense :+1: